db/database.go

```diff
@@ -87,10 +100,6 @@ func (db *Database) ClearAllOffers() error {
 	for iter.Valid() {
 		// key is the offer ID
 		key := iter.Key()
```

Comment:
You could rename the variable to `offerID` and then the comment wouldn't be needed.

Reply:
done!